From 4c999aa6e69cb1d99d4123bc7cd43b8cb27c68ba Mon Sep 17 00:00:00 2001
From: Richard Liu <r66033@freescale.com>
Date: Tue, 1 Mar 2016 13:48:45 +0800
Subject: [PATCH] MA-7665 skip build Browser on the platform without GPU 3D

Skip build Browser apk on 6sl_evk and sabresd_7d platform.
Browser apk will call webview, but webview doesn't work well
without OpenGL ES 2.0 support on android6.0, so we skip build
this apk on the platforms which don't have GPU 3D to support
OpenGL ES 2.0.

Signed-off-by: Richard Liu <r66033@freescale.com>
---
 Android.mk | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/Android.mk b/Android.mk
index 06b02d6..8dd73b3 100644
--- a/Android.mk
+++ b/Android.mk
@@ -1,3 +1,4 @@
+ifeq ($(HAVE_FSL_IMX_GPU3D),true)
 LOCAL_PATH:= $(call my-dir)
 include $(CLEAR_VARS)
 
@@ -26,3 +27,4 @@ include $(BUILD_PACKAGE)
 
 # additionally, build tests in sub-folders in a separate .apk
 include $(call all-makefiles-under,$(LOCAL_PATH))
+endif
-- 
1.8.0

